TrackletAlgorithm/VMStubMEMemoryCM.h

```diff
@@ -71,8 +71,8 @@ class VMStubMECM : public VMStubMECMBase<VMSMEType>
 
   typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEIDSize> VMSMEID;
   typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEBendSize> VMSMEBEND;
-  typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEFinePhiSize> VMSMEFINEPHI;
   typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEFineZSize> VMSMEFINEZ;
+  typedef ap_uint<VMStubMECMBase<VMSMEType>::kVMSMEFinePhiSize> VMSMEFINEPHI;
```

Comment:
Most changes in this file seems a bit redundant as things (related to finephi) have just moved around

Reply:
The functions in this memory were placed in the orderer the bits are written in the memory (index,bend,finephi,fine z/r), thus these changes don't really make sense

Reply:
I've flipped these all back to match the ordering in the master branch.